seqr/fixtures/reference_data.json

```diff
@@ -1106,7 +1106,7 @@
         "gene_description": "Tumor necrosis factor receptor superfamily, member 4",
         "comments": "mutation identified in 1 IMD16 family",
         "phenotype_inheritance": "Autosomal recessive",
-        "phenotype_mim_number": 615593,
+        "phenotype_mim_number": 615120,
```

Comment:
there is already a different fixture with this phenotype_mim_number, so this changemeans existing tests are not testing the new behavior introduced in this PR